src/ansys/fluent/core/session_solver.py

```diff
@@ -214,7 +214,7 @@ def settings(self):
                 flproxy=self._settings_service,
                 version=self._version,
                 file_transfer_service=self._file_transfer_service,
-                scheme_eval=self._fluent_connection.scheme_eval.scheme_eval,
+                scheme_eval=self._fluent_connection._connection_interface.scheme_eval.scheme_eval,
```

Comment:
I guess at session level, we have `self.scheme_eval` directly

Reply:
Scheme eval service is only created once at FluentConnection level only, right?

Reply:
yeah, but as we have added `<session>.scheme_eval` in session.py referering the same object, here we should have `self.scheme_eval`.

Reply:
Thank you @mkundu1. Got your point.